frontend/src/containers/OnBoarding.js

```diff
@@ -55,12 +54,17 @@ export class OnBoarding extends Component {
       githubUsername,
       fetchReposFromGitHub,
       appendGithubForm,
-      params
+      params,
+      notify
     } = this.props;
 
-    if (githubUsername && this.state.step === 2) {
+    if (githubUsername && this.state.step === 1) {
       appendGithubForm({username: githubUsername, token: params.token});
-      fetchReposFromGitHub(githubUsername);
+      fetchReposFromGitHub(githubUsername)
+      .catch((error) => {
+        this.setState({step: 0});
```

Comment:
The problem is that it lands you back to the first page with the notification and then you have no options. If you click on "Connect Github" again, since you are already logged in, you are back on the first page with the same message. Is there a way to avoid that?